gradyrobbins/fifty
# closes issue #39

# Description
this pr addresses issue #39

clicking on the territories at the bottom of the <Learn /> component, the images were not being rendered and instead, placeholder images were being rendered instead.



## Type of change

Please delete options that are not relevant.


# Testing Instructions for Change Made
click on "Learn more" In the navbar; click the territories at the bottom, ensure corresponding images render , rather than placeholder images of kitties.

# Checklist:
